test-cypress/steps/repository-steps.js

```diff
@@ -234,4 +254,16 @@ export class RepositorySteps {
     static getLocalGraphDBTable() {
         return cy.get('#wb-locations-locationInGetLocations');
     }
+
+    static clickObdaSelectFilesBtn() {
+       cy.get('.selectFiles').first().click();
```

Comment:
Why `first`? Provide some context to your selectors or pass a parameter and use the `eq(index)` method in cypress